00-programming-fundamentals/study-like-a-programmer.md

```diff
@@ -0,0 +1,100 @@
+# How to Study like a Programmer
+
+## Learning Goals
+
+- Have a broad overview of different layers of abstraction to study
```

Comment:
This is a _minor_ point, but I was always taught that learning goals are actions a student should be able to do or perform at the end, not things they have.

That said, your learning goals are understood.